src/foundry/foundry.js/pixi/containers/placeableObjects/token.d.ts

```diff
@@ -405,7 +337,7 @@ declare global {
      * Return a Promise that resolves to the Token once the animation for the movement has been completed
      * @param x       - The x-coordinate of the token center
      * @param y       - The y-coordinate of the token center
-     * @param animate - Animate the movement path, default is true
+     * @param animate - Animate the movement path, default is `true`
      * @returns The Token after animation has completed
      */
     setPosition(x: number, y: number, { animate }?: { animate?: boolean }): Promise<this>;
```

Comment:
might make sense to create an interface for the options type to be able to document both `options` and  `animate` properly. Also we should document the default with ``(default: `true`)``